about.html

```diff
@@ -7,65 +7,73 @@
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+    <div class="top">
+        <a href="index.html" class="logo1">
+            <img a href="index.html" class="logo" src="img/lambda-black.png" alt="Great Idea! Company logo.">
+        </a>
+            <nav>
+                <a href="#">Services</a>
+                <a href="#">Product</a>
+                <a href="#">Vision</a>
+                <a href="#">Features</a>
+                <a href="about.html">About</a>
+                <a href="contact.html">Contact</a>
+            </nav>
+            
+        </div>
+        <img class="index-header-img" src="img/jumbo-about.png" alt="City of developers">
+
+    <div class="container">
+        <div class="top-content">
+          <div class="text-container">
+            <h3>About</h3>
+            <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+          </div>
+        </div>
+        <div class="mid-content">
+          <div class="block">
+            <img src="img/about-plan.png" alt="strategy">
+            <h3>Strategy</h3>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <a href="#" class="button">Learn More</a>
+          </div>
+
+          <div class="block">
+            <img src="img/about-working.png" alt="strategy">
+            <h3>How We Work</h3>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <a href="#" class="button">Learn More</a>
```

Comment:
Anchor tags are used mostly to link an element in HTML, where href directs to the HTML or URL. Since this is a button, the best way to go about it is to give it a <button> tag.